src/extractors/custom/wired.jp/index.js

```diff
@@ -0,0 +1,41 @@
+import URL from 'url';
+
+export const WiredJpExtractor = {
+  domain: 'wired.jp',
+
+  title: {
+    selectors: ['h1.post-title'],
+  },
+
+  author: {
+    selectors: ['p[itemprop="author"]'],
+  },
+
+  date_published: {
+    selectors: ['time'],
+    format: 'YYYY.MM.DD dddd HH:mm',
+    timezone: 'Asia/Tokyo',
+  },
+
+  dek: null,
```

Comment:
I think we can use the `.post-intro` / `p[itemprop="description"]` element for this field since it does appear to serve as a dek for the article (a summary below the headline).

Reply:
Thank you.
I fixed the `dek` selector.